src/app/sales/sale.class.ts

```diff
@@ -0,0 +1,33 @@
+import { ICake } from "../cakes/cake.class";
+
+export interface ISale {
```

Comment:
Since we are using a Single File Responsibility file distribution, this Interface should live in a separate file, maybe named `sale.interface.ts`.